src/main.qml

```diff
@@ -298,98 +309,6 @@ Application {
                         anchors.centerIn: parent
                     }
                 }
-
```

Comment:
Agreed, thanks for looking into the availability of an actual currentTemp/tempBig value.
This tempBig was ment as a placeholder. I am completely fine with it being removed now that it became apparent that such a value will not be available to poll from OWM.
Thank you very much for reordering the design around that major change.

![image](https://user-images.githubusercontent.com/15074193/162804645-9127e9bc-9ad6-4f4a-b523-80ded89f2ff8.png)

I have tested on different screen size watches now and fully approve the overall visual change.